default.go

```diff
@@ -119,7 +119,7 @@ func Trace(ctx context.Context, msg string, params ...interface{}) {
 // case the severity will be inferred from the error.
 func FromError(ctx context.Context, msg string, err error, params ...interface{}) {
 	if l := DefaultLogger(); l != nil {
-		params = []interface{}{err, params}
+		params = append([]interface{}{err}, params...)
 		if ll, ok := l.(FromErrorLogger); ok {
 			ll.FromError(ctx, msg, err, params...)
```

Comment:
I know this isn't directly related to your change, but it seems odd to me to be including the `err` in the `params` at all, when it's also passed explicitly to `ll.FromError`. Could we just not include it in `params`?

Reply:
I agree and have updated. If the logger implements the FromErrorLogger interface then they take in the error so there's no need to prepend. However, if the logger doesn't implement it then we prepend it so that the behaviour is similar to if you `slog.Error(ctx, "some error: %v", context.Canceled, "more params")` (the error is a param)